wm/config.go

```diff
@@ -120,13 +120,15 @@ func (conf *Configuration) loadMouseConfigSection(
 	for _, key := range cdata.Keys(section) {
 		mouseStr := key.Name()
 		for _, cmd := range key.Strings() {
+			logger.Warning.Printf("mouse cmd is %v", cmd)
 			if _, ok := conf.mouse[ident]; !ok {
 				conf.mouse[ident] = make([]mouseCommand, 0)
 			}
 
 			if err := gribbleEnv.Check(cmd); err != nil {
 				logger.Warning.Printf(
 					"Could not parse command '%s' because: %s", cmd, err)
+				panic("fuck")
```

Comment:
Something more descriptive please =P

Reply:
do you want to change the warning printf into some kind of fatal instead?

Reply:
Seems a good case to use a kind of `log.Fatalf` here.